.template-env

```diff
@@ -14,3 +14,6 @@ NB_GRAPH_IMG=stardog/stardog:8.2.2-java11-preview
 # NB_GRAPH_ADDRESS=graph
 # NB_GRAPH_PORT_HOST=5820
 # NB_GRAPH_PORT=5820
+# NB_QUERY_TAG=latest
+# NB_QUERY_PORT_HOST=3000
```

Comment:
damn, good catch!

Reply:
Yeah, I can confirm that `API_QUERY_URL` has to include a trailing `/` slash for the request to succeed

Reply:
Just mental note for me. We need to

- Add the API query URL to template and compose file as suggested
- Add the .env file to docker ignore to avoid Image build context leaking into public docker image
- Ensure the ci build process for the public docker image starts from a clean context (e.g. separate job)
- Document that the API query URL has to end in forward slash
- Put some sort of check into the query tool for when there is no tailing slash - and then add one
- Make the port for the API configurable as suggested